ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/BadBlockManager.java

```diff
@@ -49,6 +49,12 @@ public void addBadBlock(final Block badBlock, final Optional<Throwable> cause) {
     }
   }
 
+  public void reset() {
+    this.badBlocks.invalidateAll();
+    this.badHeaders.invalidateAll();
+    this.latestValidHashes.invalidateAll();
+  }
+
```

Comment:
Where is this called?

Reply:
https://github.com/hyperledger/besu/pull/4875/files#diff-cc598aa8b256f675dfcbd331db0955fd273916856708992c92364aa373405f12R49